### ISSUE TITLE
bodhi-client cannot handle expiring sessions

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#1474]
The bodhi client cannot work with expiring sessions. Once its session expires, it will never attempt to get a new session but will instead assume it can use the existing session. Thus, it does not matter if the user types the correct password or not:

```
$ time fedpkg update
Password:
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/bodhi/client/__init__.py", line 132, in new
    resp = client.save(**update)
  File "/usr/lib/python2.7/site-packages/bodhi/client/bindings.py", line 66, in wrapper
    result = method(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/bodhi/client/bindings.py", line 161, in save
    data=kwargs)
  File "/usr/lib/python2.7/site-packages/fedora/client/openidbaseclient.py", line 254, in send_request
    'output': to_bytes(output.text),
ServerError: ServerError(https://bodhi.fedoraproject.org/updates/, 403, Error returned from json module while processing https://bodhi.fedoraproject.org/updates/: Expecting value: line 6 column 1 (char 5)
```

# Workaround

Removing your cached sessions will work around the problem:

```
$ rm ~/.fedora/openidbaseclient-sessions.cache
```

Note that this workaround will only work until your new session expires, and then you will hit this problem again (and you can ```rm``` again to workaround once more). Fedora's Bodhi instance is configured to expire sessions after 24 hours.

**Comment by bowlofeggs** [comment]
Since this bug will be fixed in python-fedora, I am going to kick it off the 2.6.1 milestone and go ahead and release 2.6.1.

**Comment by bowlofeggs** [comment]
This fix for this has been cherry picked to the 2.6 branch as b59a27bc1737887eb4b815e88cec66f276ba2b4d.
